akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -62,5 +65,30 @@ class AskSpec extends TestKit("AskSpec") with TypedAkkaSpec with ScalaFutures {
       val answer = noSuchActor ? Foo("bar")
       answer.recover { case _: AskTimeoutException ⇒ "ask" }.futureValue should ===("ask")
     }
+
+    "must transform a replied akka.actor.Status.Failure to a failed future" in {
+      implicit val untypedSystem = akka.actor.ActorSystem("AskSpec-untyped-1")
+      try {
+        // to be honest I'm not too sure this would actually ever happen
```

Comment:
what does this comment mean?

Reply:
It struck me while writing this test: The responding untyped actor would have to cast the incoming `ActorRef[Protocol]` to `ActorRef[akka.actor.Status]` before replying with that `Failure`, when would that ever happen?